tool/teleport/common/teleport.go

```diff
@@ -125,6 +125,15 @@ func Run(options Options) (executedCommand string, conf *service.Config) {
 		"Start Teleport in FedRAMP/FIPS 140-2 mode.").
 		Default("false").
 		BoolVar(&ccf.FIPS)
+	start.Flag("app_name",
```

Comment:
Just noticed that these flag names are inconsistent with the rest of the multi-word flags which use dashes instead of underscores, so I would consider renaming to `app-name`, `app-uri` and `app-public-addr`.

Reply:
Also, I see this in the `teleport start --help` output:

```
Notes:
  --roles=node,proxy,auth

  This flag tells Teleport which services to run. By default it runs all three. 
  In a production environment you may want to separate them.
```

Should the `--roles` flag description be updated to include the `app` role as well?

Reply:
@benarent What do you think?